src/utils/protostar_directory.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+from typing import Optional, Union
+
+import tomli
+from packaging import version
+from packaging.version import LegacyVersion
+from packaging.version import Version as PackagingVersion
+
+from src.protostar_exception import ProtostarException
+
+
+class ProtostarDirectory:
+    class ProtostarNotInstalledException(ProtostarException):
```

Comment:
Are we using it anywhere?

Reply:
In which scenario user could run protostar without protostar installed? Because Im not sure

Reply:
This is a leftover. Previously, I used `os.which('protostar')` to find the installation directory.